operator/pkg/validate/common_test.go

```diff
@@ -0,0 +1,231 @@
+package validate
+
+import (
+	"regexp"
+	"testing"
+
+	"istio.io/istio/operator/pkg/util"
+)
+
+func Test_validatePortNumberString(t *testing.T) {
```

Comment:
These functions are tested in the top level validate function, and are too specific tests which I think won't improve the test coverage?

Reply:
See https://github.com/istio/istio/blob/master/operator/pkg/validate/validate_test.go. I think enriching the validate test would make more sense.

Reply:
But `DefaultValidations` doesn't validate these functions, or should `DefaultValidations` be modified?

Reply:
See https://github.com/istio/istio/blob/a04ccfde0f984dd0b17a4298acdddb30c0e850f8/operator/pkg/validate/validate_values.go#L26-L33